app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :user, foreign_key: 'users_id'
+  belongs_to :post, foreign_key: 'posts_id'
+  after_save :update_likes_counter
+
+  def update_likes_counter
+    post.increment!(likes_counter)
+  end
```

Comment:
- [ ] Kindly put colon `:` before likes_counter because likes_counter is an attribute in the posts table so to get an attribute we always add a colon `:` at the start of the attribute in this case it will look like this **(:likes_counter)** :blush: